src/main/resources/application.yml

```
@@ -1,53 +1,54 @@
-    presigned-expiration-seconds: 300
-
-logging:
-  level:
-    org.hibernate.SQL: DEBUG
-    org.hibernate.type.descriptor.sql.BasicBinder: TRACE
-
-
-server:
-  port: 8081
\ No newline at end of file
+spring:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)